src/app/modules/group/components/user-list/user-list.component.ts

```diff
@@ -8,43 +10,48 @@ import { GetGroupMembersService, Member } from '../../http-services/get-group-me
   templateUrl: './user-list.component.html',
   styleUrls: [ './user-list.component.scss' ]
 })
-export class UserListComponent implements OnChanges, OnDestroy {
+export class UserListComponent implements OnInit {
```

Comment:
So you do not handle the change of group anymore.

Reply:
fixed